sdks/uniswapx-sdk/src/order/PriorityOrder.ts

```diff
@@ -241,17 +288,9 @@ export class PriorityOrder implements OffChainOrder {
    * Returns the resolved order with the given options
    * @return The resolved order
    */
-  resolve(options: PriorityOrderResolutionOptions): ResolvedUniswapXOrder {
-    if (options.currentBlock && options.currentBlock.lt(this.info.startBlock)) {
-      throw new OrderNotFillable("Start block in the future");
-    }
-    return {
-      input: {
-        token: this.info.input.token,
-        amount: scaleInput(this.info.input, options.priorityFee),
-      },
-      outputs: scaleOutputs(this.info.outputs, options.priorityFee),
-    };
+  resolve(_options: PriorityOrderResolutionOptions): ResolvedUniswapXOrder {
+    // no cosigner data so no resolution possible
+    throw new Error("Method not implemented.");
```

Comment:
Should still be able to resolve though, can check auctionStartBlock and return scaled input/output numbers?

Reply:
Yeah I guess it's more a matter of should we resolve an unsigned order... given that without the `targetBlock`, the resolved numbers will most likely be different from the full order numbers?

Reply:
yea I agree and I see why we don't resolve them for Xv2 since the input/output amounts are changing, though in this case the important order numbers (input and output required) don't really change if the cosigner is there or not?